themes/bootstrap3/templates/myresearch/bulk-action-buttons.phtml

```diff
@@ -2,23 +2,29 @@
   <input type="hidden" name="listID" value="<?=$this->escapeHtmlAttr($list->id)?>">
   <input type="hidden" name="listName" value="<?=$this->escapeHtmlAttr($list->title)?>">
 <?php endif; ?>
-<?php $user = $this->auth()->isLoggedIn(); ?>
+<?php
+$user = $this->auth()->isLoggedIn();
+$actionLimits = $this->config()->get('config')?->BulkActions?->limits;
```

Comment:
If we add more smarts to limits as suggested above, we'll probably need more view helper logic somewhere to apply equivalent behavior here. If you want to go that route, I can give it a little more thought and propose a location, if you don't already have ideas.

Reply:
I added a view helper for rendering bulk action buttons that checks the limits.